cmake/iOS.cmake

```diff
@@ -0,0 +1,208 @@
+# This file is based off of the Platform/Darwin.cmake and Platform/UnixPaths.cmake
```

Comment:
We will need to update the LICENSE as this is a derived work/source inclusion.  At least that is my interpretation of this comment.  Please let me know if that is not the case.